### DISCUSSION TITLE
How can i use ImageMagick to convert a image from  PCI-p3 to srgb？

### DISCUSSION THREAD

**Discussion by ThenMorning** [ImageMagick/ImageMagick#3264] [Help]
I can‘t find any method about 'color space'
Thanks!

**Comment by snibgo** [comment]
What version of IM, on what platform?

Does the input file contain an embedded ICC profile? "magick in.p3 -verbose info:" will tell you.

I suggest converting with the "-profile" operation. If the input has no profile, first assign the appropriate profile. Then convert it to a sRGB profile.

**Reply by ThenMorning** [reply]
Thanks for help!
I want to recognize the color on the image and find the color space may effect recognize result.
Platform: linux
Does the input file contain an embedded ICC profile?   No,just a picture.
